app/recipe/migrations/0002_alter_ingredient_recipe.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.0.10 on 2023-05-24 15:09
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('recipe', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='ingredient',
+            name='recipe',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='ingredients', to='recipe.recipe'),
+        ),
+    ]
```

Comment:
You have a migration but I can't see that you've updated the model. Maybe that was done before you created the PR?